### PR TITLE
Add patches

### PR DESCRIPTION
**Pull request by ocefpaf**
Partially addresses https://github.com/conda-forge/pypy3.6-feedstock/issues/15

cc: @isuruf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,54 +7,7 @@ Package license: MIT
...
-
-```
-pip3 install ...
-```
-
-(You can verify that it's the correct `pip3`/`pip` using `which {pip3,pip}`. It should be something like `.../miniconda3/envs/pypyenv/bin/pip` )
-
-🔴 You CANNOT use `conda install x` to install anything,
-because Conda provides pre-built package for the regular CPython interpreter, which are incompatible with PyPy.
+Summary: PyPy is a Python interpreter and just-in-time compiler.
```
Comment (ohadravid) [inline README.md]: Don’t let the rerender edit this part of the file :)

Comment (ocefpaf) [reply README.md]: That is kind of against what we expect of the rerender. Maybe we should document that elsewhere that won't be overwritten automatically.

Comment (isuruf) [reply README.md]: The text there is not true anymore. See https://conda-forge.org/blog//2020/03/10/pypy

Comment (ohadravid) [reply README.md]: Awesome! I guess we don’t need this text anymore!

**Pull request merged**